docs/src/main/sphinx/connector/hive.rst

```diff
@@ -997,9 +997,9 @@ Hive connector limitations
 --------------------------
 
 * :doc:`/sql/alter-schema` usage fails, since the Hive metastore does not support renaming schemas.
-* :doc:`/sql/delete` applied to non-transactional tables is only supported if the ``WHERE`` clause matches entire partitions.
-  Transactional Hive tables with format ORC support "row-by-row" deletion, in which the ``WHERE`` clause may match arbitrary
-  sets of rows.
+* :doc:`/sql/delete` applied to non-transactional tables is only supported if the table is partitioned and
```

Comment:
While you are changing this paragraph I suggest to just rewrap it to 80 char width.